docs/guides/configuration-guide/loadbalancer.md

```diff
@@ -25,42 +29,76 @@ hosts_additional_entries:
 
 ## TLS certificates
 
-To enable external TLS encryption:
+:::warning
+To avoid unnecessary additional work and problems, it is recommended that you configure TLS with the intended target
+configuration of the specific environment during the initial rollout process.
+:::
 
-```yaml title="environments/kolla/configuration.yml"
-kolla_enable_tls_external: "yes"
-```
+For example: Changes to the configuration of TLS or the fully qualified domain names (FQDNs) will result in new URLs (with and without the https prefix)
+stored in the Openstack databases, and in the case of self-signed certificates, the CA certificate must be distributed to all participating clients.
 
-To enable internal TLS encryption:
+As a result, the involved Ansible Plays must at least be executed in the correct order, and not all Ansible Plays can handle all possible configuration transitions on their own.
+Therefore, in some cases, manual adjustments must be made to the systems.
```

Comment:
how could the operator run this in a wrong order? ansible plays in general need to be executed in the correct order, this is not really a specific property of setting up TLS in osism? Also _if_ we mention this at all, what is the correct order to execute the ansible plays? It's not mentioned here?